model_compression_toolkit/core/common/hessian/trace_hessian_calculator.py

```diff
@@ -75,3 +75,24 @@ def compute(self) -> List[float]:
         computation method for the trace Hessian approximation.
         """
         raise NotImplemented(f'{self.__class__.__name__} have to implement compute method.')  # pragma: no cover
+
+    def _unfold_outputs(self, outputs: Any) -> List[Any]:
```

Comment:
Where would you think it should be? Since it's may use other parts outside the hessian package, and we don't have a utils module (if I'm not wrong)

Reply:
I meant the name and the variables in this function, not the location. Because its functionality is unfolding: taking tensor and return list the using the names "output" in this function is not neccecry. For example the input can be "input_tensor" instead of "outputs".